src/pages/ProductList/common/ListSidebar.js

```diff
@@ -0,0 +1,219 @@
+import React, { useEffect, useState } from 'react';
+import FlexibleBtn from '../components/FlexibleBtn';
+import styled from 'styled-components';
+import {
+  faWonSign,
+  faLocationDot,
+  faStar,
+  faMap,
+  faFilter,
+  faList,
+} from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { motion } from 'framer-motion';
+
+const ListSidebar = props => {
+  const { hotelData, setView } = props;
```

Comment:
```suggestion
const ListSidebar = ({ hotelData, setView }) => {
```
* 인자 자리에서 바로 구조분해 할당 하면 한줄 더 줄일 수 있습니다

Reply:
감사합니다!